content/en/docs/reference/glossary/userns.md

```diff
@@ -0,0 +1,23 @@
+---
+title: user namespace
+id: userns
+date: 2021-07-13
+full_link: https://man7.org/linux/man-pages/man7/user_namespaces.7.html
+short_description: >
+  A kernel feature to emulate root. Used for "rootless containers".
+
+aka:
+tags:
+- security
+---
+
+A kernel feature to emulate root. Used for "rootless containers".
+
+<!--more-->
+
+User namespace is a Linux kernel feature that allows a non-root user to emulate fake root privileges that are enough to
```

Comment:
this glossary definition is optimized for the specific feature. We plan also to work on other use case involving user namespaces - see https://kinvolk.io/blog/2020/12/improving-kubernetes-and-container-security-with-user-namespaces/ for explainer. This glossary item is fine for now, but needs to be updated with the upcoming work. Perhaps insert the TODO on the page?

Reply:
added